### PR TITLE
Reformat project, rename objects and convert 'tag' into a group

### LINKED ISSUE #33: Use sub-command instead of underscore
e.g. `?tag add` over `?tag_add`

### PR DESCRIPTION
**Pull request by Pythonic-Rainbow**
Resolves #33
Renamed  various objects

### CODE REVIEW

**Review by sbhadr — APPROVED** [review]
Was this tested?

**Comment by Pythonic-Rainbow** [comment]
image

Looks good on my side.

File: cogs/tagging.py [hunk 1]
```diff
@@ -48,24 +48,25 @@ async def tag_add(self, ctx, name, *, content: str):
 
         db.taggingEntry.insert(ctx.message.channel.id, name, content, ctx.message.created_at)
         await ctx.send(f'Tag "{name}" created.')
-    
-    @commands.command()
+
+    @tag.command()
     @commands.has_role("Library Developer")
```
Comment (Pythonic-Rainbow) [inline cogs/tagging.py:53]: Actually, why do we need to check if the sender is a lib dev here?

Comment (sbhadr) [reply cogs/tagging.py:53]: Editing & creating tags are only for library developers. Fetching tags is for anyone.

**Comment by sbhadr** [comment]
Did you double-check to see if the ``data.py`` changes work?

**Comment by Pythonic-Rainbow** [comment]
Yes. All I did was capitalising the classes.

**Review by sbhadr — APPROVED** [review]

**Pull request merged**